tests/components/linear_garage_door/test_config_flow.py

```diff
@@ -0,0 +1,128 @@
+"""Test the Linear Garage Door config flow."""
+
+from unittest.mock import patch
+
+from linear_garage_door.errors import InvalidDeviceIDError, InvalidLoginError
+
+from homeassistant import config_entries
+from homeassistant.components.linear_garage_door.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == FlowResultType.FORM
+    assert result["errors"] is None
+
+    with patch(
+        "linear_garage_door.Linear.login",
```

Comment:
You should patch locally instead of globally.

"homeassistant.components.linear_garage_door.Linear"

Reply:
do this for all of your patches